client/src/App.scss

```diff
@@ -4,7 +4,7 @@
 main > article {
 	padding: calc(#{layout.$baseline} * 4);
 
-	> h2 {
+	> h2 :last-child {
```

Comment:
is might be difficult to ready for some I think we can have class name for span inside h2 like` <span classname="resourceTitle"></span>`

and in Resource folder in index.scss we can have class resourceTitle so we can remove this syle 

	```
> h2 :last-child {
		border-bottom: 2px solid palette.$red;
		display: inline-block;
		padding-bottom: calc(#{layout.$baseline} / 2);
		background-color: red;
	}

```
app.scss